api/app/Repositories/SecondTestRepository.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: redjik
+ * Date: 15.07.15
+ * Time: 2:36
+ */
+
+namespace App\Repositories;
+
+use App\Models\BaseModel;
+use App\Models\SecondTestEntity;
+
+class SecondTestRepository extends BaseRepository
+{
+    /**
+     * Model name.
+     *
+     * @return BaseModel
+     */
+    protected function model()
+    {
+        return new SecondTestEntity();
```

Comment:
Can we have the `BaseRepository` define a `model()` and a `setModel($modelClass)`method that so that the controller can use the `BaseRepository` in the constructor and pass the model name if there is no need to extend the `BaseRepository` methods?